src/glossary.md

```diff
@@ -8,14 +8,21 @@ the structure of the program when the compiler is compiling it.
 ### Arity
 
 Arity refers to the number of arguments a function or operation takes.
-For example, `(2, 3)` and `(4, 6)` have arity 2, and`(8, 2, 6)` has arity 3.
+For example, `f(2, 3)` and `g(4, 6)` have arity 2, while `h(8, 2, 6)` has arity 3.
 
 ### Array
 
 An array, sometimes also called a fixed-size array or an inline array, is a value
 describing a collection of elements, each selected by an index that can be computed
 at run time by the program. It occupies a contiguous region of memory.
 
+### Associated Item
+
+An associated item is an item that is associated with another item. Associated
+items are defined in [implementations] and [traits]. Only functions, constants,
+and types can be associated.
```

Comment:
Hmm, are types associated, or are type aliases associated?  You can't put an `enum Foo {}` in an `impl` to satisfy a `type Foo;` in the trait, right?

Is it worth distinguishing between associated items are "declared in traits" and "defined in impls"?  The declared/defined words are used in trait impls below.